subgraphs/rari-yag-v1/src/common/utils/constants.ts

```diff
@@ -0,0 +1,230 @@
+  export const DYNAMIC_PROTOCOL_FEE = "DYNAMIC_PROTOCOL_FEE";
+}
+
+export namespace RewardTokenType {
+  export const DEPOSIT = "DEPOSIT";
+  export const BORROW = "BORROW";
+}
+
+export namespace LendingType {
+  export const CDP = "CDP";
+  export const POOLED = "POOLED";
+}
+
+export namespace RiskType {
+  export const GLOBAL = "GLOBAL";
+  export const ISOLATED = "ISOLATED";
+}
+
+export namespace InterestRateType {
+  export const STABLE = "STABLE";
+  export const VARIABLE = "VARIABLE";
+  export const FIXED_TERM = "FIXED_TERM";
+}
+
+export namespace InterestRateSide {
+  export const LENDER = "LENDER";
+  export const BORROWER = "BORROWER";
+}
+
+export namespace TransactionType {
+  export const DEPOSIT = "DEPOSIT";
+  export const WITHDRAW = "WITHDRAW";
+}
+
+export namespace RariPool {
+  export const YIELD_POOL = "Yield Pool";
+  export const STABLE_POOL = "Stable Pool"; // USDC and DAI
+  export const ETHER_POOL = "Ether Pool";
+}
+
+//////////////////////////////
+///// Ethereum Addresses /////
+//////////////////////////////
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+
+export const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+export const ETH_NAME = "Ether";
+export const ETH_SYMBOL = "ETH";
+
+////////////////////////
+///// Type Helpers /////
+////////////////////////
+
+export const DEFAULT_DECIMALS = 18;
+
+export const USDC_DECIMALS = 6;
+export const USDC_DENOMINATOR = BigDecimal.fromString("1000000");
+
+export const BIGINT_ZERO = BigInt.fromI32(0);
+export const BIGINT_ONE = BigInt.fromI32(1);
+export const BIGINT_TWO = BigInt.fromI32(2);
+export const BIGINT_THOUSAND = BigInt.fromI32(1000);
+export const BIGINT_MAX = BigInt.fromString(
+  "115792089237316195423570985008687907853269984665640564039457584007913129639935",
+);
+
+export const INT_ZERO = BIGINT_ZERO.toI32();
+export const INT_ONE = BIGINT_ONE.toI32();
+export const INT_TWO = BIGINT_TWO.toI32();
+
+export const BIGDECIMAL_ZERO = new BigDecimal(BIGINT_ZERO);
+export const BIGDECIMAL_ONE = new BigDecimal(BIGINT_ONE);
+export const BIGDECIMAL_TWO = new BigDecimal(BIGINT_TWO);
+
+export const MAX_UINT = BigInt.fromI32(2).times(BigInt.fromI32(255));
+
+/////////////////////
+///// Date/Time /////
+/////////////////////
+
+export const SECONDS_PER_HOUR = 60 * 60; // 3600
+export const SECONDS_PER_DAY = 60 * 60 * 24; // 86400
+export const MS_PER_DAY = new BigDecimal(BigInt.fromI32(24 * 60 * 60 * 1000));
+export const DAYS_PER_YEAR = new BigDecimal(BigInt.fromI32(365));
+export const MS_PER_YEAR = DAYS_PER_YEAR.times(new BigDecimal(BigInt.fromI32(24 * 60 * 60 * 1000)));
+export const BLOCKS_PER_YEAR = BigDecimal.fromString("2102400");
+
+///////////////////////////
+//// Rari Input Tokens ////
+///////////////////////////
+
+// Note: Deposit/Withdrawal events return the currency as an indexed string
+// so we need to create a mapping between the 'currencyCode' to Token
+
```

Comment:
do you know how the indexed string is created/hashed?

Reply:
added commenting, see https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378 for more info